WebSocketTest/Decoders/JsonDecoder.cs

```diff
@@ -1,12 +1,17 @@
 ﻿using WebSocketTest.Datatypes;
+using System.Text.Json;
+//using Newtonsoft.Json;
+
 
 namespace WebSocketTest.Decoders
 {
 	static class JsonDecoder
 	{
 		static public Inputs ParseToInputs(Inputs target)
 		{
-			return new Inputs();
+            string json = JsonSerializer.Serialize(target);
+            var convertedObject = JsonSerializer.Deserialize<Inputs>(json); // Converts json string to Inputs object 
```

Comment:
cant you just return `return JsonSerializer.Deserialize<Inputs>(json);`
Also, comments on the previous line instead of the same